### PR TITLE
Apply branding guidelines on  `compress-diskimg.sh`

### PR DESCRIPTION
**Pull request by sparkles-laurel**
small changes such as fixing capitalisation of the word `popcorn`

### CODE REVIEW

**Comment by sparkles-laurel** [comment]
do you want me to do a narcissistic merge?

File: compress-diskimg.sh [hunk 1]
```diff
@@ -50,5 +50,5 @@ fi
 echo -e "\e[1m[INFO]\e[0m Compressed disk image successfully, written to \"target/x86_64-arch/debug/bootimage-popcorn.bin.zst\"";
 
 # i mean those who work on popcorn must be nocturnal people so...
-echo -e "\e[1m[INFO]\e[0m Popcorn project wishes you a nice day!";
+echo -e "\e[1m[INFO]\e[0m PURE/popcorn project wishes you a nice day!";
```
Comment (electron271) [inline compress-diskimg.sh]: only popcorn as this is the kernel compressor not related to user space

**Review by electron271 — CHANGES_REQUESTED** [review]

File: compress-diskimg.sh (hunk 1 above)
Comment (sparkles-laurel) [reply compress-diskimg.sh]: i meant the entire project's brand is that

Comment (electron271) [reply compress-diskimg.sh]: still, id say just put only popcorn there,.makes more sense

Comment (sparkles-laurel) [reply compress-diskimg.sh]: i did that

**Review by electron271 — APPROVED** [review]
yeah sure

**Pull request merged**